File: RightmostChar/py3/right.py
# Rightmost Char solution in Python 3 for CodeEval.com by Steven A. Dunn

import sys

for line in open(sys.argv[1], 'r'):
  if not line: continue
  params = line.rstrip('\n').split(',')
  print (params[0].rfind(params[1]))

File: RightmostChar/Ruby/right.rb
# rightmost char soln in rb for codeeval.com by steven a dunn

File.open(ARGV[0], 'r').each_line do |line|
    params = line.chomp.split(',')
    idx = params[0].rindex(params[1])
    if idx.nil?
        puts -1
    else
        puts idx
    end
end